doc/nrf/known_issues.rst

```diff
@@ -2691,6 +2691,13 @@ NCSDK-17501: Partition Manager: Ignored alignment for partitions
 
   **Workaround:** Edit the affected configurations file so that ``align`` is correctly placed inside the ``placement`` section of the config file.
 
+.. rst-class:: v2-2-0
+
+NCSDK-19536: Boards without a uart1 node do not compile with TF-M logging enabled
```

Comment:
```suggestion
NCSDK-19536: Boards without a UART1 node do not compile with TF-M logging enabled
```

Reply:
Oh, I remembered it the other way around. Then ignore that, in the next line you have UART1 with capital letters, maybe we can convert that to lower case?